src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/specification/UserSpecification.kt

```diff
@@ -151,14 +164,24 @@ class UserSpecification(private val filter: UserFilter) : Specification<UserPers
     }
 
     filter.activeCaseloadId?.run {
-      predicates.add(activeCaseload(this))
+      if (filter.showOnlyLSAs == false) {
+        predicates.add(activeCaseload(this))
+      }
+    }
+
+    filter.showOnlyLSAs.takeIf { it == true }?.run {
```

Comment:
Might be nicer to just replace with
```if (filter.showOnlyLSAs) {```

Reply:
changed to  if (filter.showOnlyLSAs == true)